nautobot/dcim/models/device_component_templates.py

```diff
@@ -34,7 +34,7 @@
 )
 
 
-class ComponentTemplateModel(BaseModel, CustomFieldModel, RelationshipModel):
+class ComponentTemplateModel(BaseModel, CustomFieldModel, RelationshipModel, ComputedFieldModelMixin):
```

Comment:
Mixins should always precede bases (on the left).

Reply:
Again here, just being consistent with the way things are set up now with the other abstract models (see `CustomFieldModel` and `RelationshipModel`). I can rename `ComputedFieldModelMixin` to `ComputedFieldModel` if that helps?